1_fetch/src/get_nlcd_LC.R

```diff
@@ -120,38 +120,48 @@ read_subset_LC_data <- function(LC_data_folder_path,
   
   # Loop through sub-folders, combine datasets, and subset through Join
   for(LC_data in LC_data_folder_path){
+    # Check that downloaded data exists in folder 
+    if(length(list.files(LC_data)) == 0){
+      stop(paste0('No NLCD LC data in ', LC_data,'. Please move the NLCD .txt files to this location.'))
+    }
```

Comment:
This check should go before the loop.
```suggestion
  # Check that downloaded data exists in folder 
    if(length(list.files(LC_data_folder_path)) == 0){
      stop(paste0('No NLCD LC data in ', LC_data_folder_path,'. Please move the NLCD .txt files to this location.'))
    }
  
  # Loop through sub-folders, combine datasets, and subset through Join
  for(LC_data in LC_data_folder_path){
```

Reply:
WHy do you prefer it not in the loop? 

I chose to place it at the beginning of the for loop because a vector of paths to downloaded data files can be passed into the `read_subset_LC_data()` argument: `LC_data_folder_path`. So if it's inside the for loop it checks the existence of the downloaded files within each element in the vector. This is what happens with target `p1_NLCD2011_data` as 4 NLCD data folders are passed here ('ImperviousnessPct_2011', 'Imperviousness100m_RipZone',   'TreeCanopy_100mBuffered_RipZone', 'NLCD_LandCover_50m_RipZone' ) 

That said, passing a vector of folder paths into list.files() works, it basically prints all the files within all the paths. 
But if one folder path within the vector has no files, the check would not stop for that specific path.

Reply:
Oh, okay, that makes sense. I missed that it's checking for each file. 
```suggestion
  
  # Loop through sub-folders, combine datasets, and subset through Join
  for(LC_data in LC_data_folder_path){
    # Check that downloaded data exists in folder 
    if(length(list.files(LC_data)) == 0){
      stop(paste0('No NLCD LC data for file ', LC_data,'. Please move the NLCD .txt file to this location.'))
    }
```